code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -110,7 +110,7 @@
 	M.Paralyze(paralyze_time)
 
 	step_to(X, M)
-	X.stop_movement()
+	X.Immobilize(X.xeno_caste.charge_type == CHARGE_TYPE_SMALL ? 0.5 SECONDS : 1.5 SECONDS)
```

Comment:
noooo please
trying to get rid of charge_type dont add more stuff that uses it!
move it to like `/datum/action/xeno_action/activable/pounce/var/pounce_immobilize_time`